presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/ShardCleaner.java

```diff
@@ -275,9 +267,35 @@ private synchronized void runBackupCleanup()
         }
     }
 
+    private synchronized void waitJitterTime()
+    {
+        try {
+            // jitter to avoid overloading database
+            long interval = this.localCleanerInterval.roundTo(SECONDS);
+            SECONDS.sleep(ThreadLocalRandom.current().nextLong(1, interval));
+        }
+        catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+        }
+    }
+
+    private synchronized void runLocalCleanupAtStartup()
```

Comment:
think about exporting this method for jmx, but without wait time. Something like "force clean".